app/admin/dashboard.rb

```diff
@@ -163,5 +163,40 @@
         end
       end
     end
+
+    columns do
+      columns do
```

Comment:
Do we need`columns` nested twice here? :)

Reply:
Fixed